components/dashboard/src/data/featureflag-query.ts

```diff
@@ -27,8 +27,7 @@ const featureFlags = {
     newProjectIncrementalRepoSearchBBS: false,
     repositoryFinderSearch: false,
     createProjectModal: false,
-    repoConfigListAndDetail: false,
-    showRepoConfigMenuItem: false,
+    configurationsAndPrebuilds: false,
```

Comment:
We may need to enable flags for customers that already have those two removed flags enabled before deploy changes to their cells

Reply:
And let's make it a function in this file like dataops ()

Reply:
Made sure the two are aligned with the new one 👌